samples/Sample.RabbitMQ.MySql/Sample.RabbitMQ.MySql.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Dapper" Version="2.0.35" />
```

Comment:
Can this reference be removed? It doesn't seem to be used

Reply:
![image](https://user-images.githubusercontent.com/12774053/84354180-00614a00-abf3-11ea-98da-c9a7e0c8e371.png)

Reply:
Thank you for pointing, can we modify the example to use ado.net?

Reply:
Yes, of course。However, keeping the dapper here just shows that multiple ORM frameworks can be supported。it can also add an example of ado.net

Reply:
> Yes, of course。However, keeping the dapper here just shows that multiple ORM frameworks can be supported。it can also add an example of ado.net

For class library projects, keeping the intrusion to a minimum is more important for copying code

Reply:
> Yes, of course。However, keeping the dapper here just shows that multiple ORM frameworks can be supported。it can also add an example of ado.net

For class library projects, keeping the intrusion to a minimum is more important for copying code